src/main/java/edu/kis/powp/jobs2d/command/manager/HistoryCommandItem.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+public class HistoryCommandItem {
+
+    String name;
+    String date;
```

Comment:
History is not only about names and dates of commands. You should also keep a copy of orignal command so we can set it as current!